app/views/users/edit_password.html.erb

```diff
@@ -0,0 +1,20 @@
+<% provide :title, t("words.change_password") %>
+<h1><%= t "words.change_password" %></h1>
```

Comment:
dùng fuzzy search đi The Fakest, thêm rule edit_password, là được chứ thêm cái words mới kia làm gì (yaoming)